site/_docs/troubleshooting.md

```diff
@@ -60,12 +60,43 @@ which may allow you to install native gems using this command (again using
 sudo gem install jekyll
 {% endhighlight %}
 
-Note that upgrading MacOS X does not automatically upgrade Xcode itself
+Note that upgrading Mac OS X does not automatically upgrade Xcode itself
 (that can be done separately via the App Store), and having an out-of-date
 Xcode.app can interfere with the command line tools downloaded above. If
 you run into this issue, upgrade Xcode and install the upgraded Command
 Line Tools.
 
+### Jekyll &amp; Mac OS X 10.11
+
+With the introduction of System Integrity Protection, several directories
+that were previously writable are now considered system locations and are no
+longer available. As a result, it is recommended that you choose one of a
+number of available Ruby environments ([RVM][], [rbenv][], [chruby][], etc.) in
+which to install Jekyll.
+
+[RVM]: https://rvm.io
+[rbenv]: http://rbenv.org
+[chruby]: https://github.com/postmodern/chruby
+
+If you elect to use a method other than Homebrew to install Ruby, it may be
```

Comment:
I realized almost as soon as I wrote the words that something just didn't sound right. Homebrew is mentioned without really any introduction or explanation at all. My thinking came out of @chmaynard's [comment][] and a follow up I added. While I think most people who get this far likely would understand it, I agree it's really not supported well and needs to be rephrased.

To your second point, that seems like a much clearer transition - I'll give it some thought and see how this can be incorporated...

[comment]: https://github.com/jekyll/jekyll/issues/3984#issuecomment-143149777